src/Date.php

```diff
@@ -122,51 +121,51 @@ public function __get($name)
 		switch ($name)
 		{
 			case 'daysinmonth':
-				$value = $this->format('t', true);
```

Comment:
Yes.  The second param in the old format method basically decided whether to convert the time to GMT or not (hence me renaming it from `formatLocal` to `formatGmt`).  If false, it was converted.